code/datums/components/food_stuff.dm

```diff
@@ -0,0 +1,102 @@
+/datum/component/consume
+/datum/component/consume/Initialize()
+	if(!istype(parent, /obj/item))
+		return COMPONENT_INCOMPATIBLE
+
+/datum/component/consume/organpoints
+	var/list/organ2points = list(/obj/item/organ/head=2,/obj/item/skull=0,/obj/item/organ/brain=3,/obj/item/organ/chest=5,/obj/item/organ/heart=2,/obj/item/organ/appendix=0,/obj/item/clothing/head/butt=0)
+
+/datum/component/consume/organpoints/Initialize()
+	RegisterSignal(parent, list(COMSIG_ITEM_CONSUMED), .proc/eat_organ_get_points)
```

Comment:
Should check parent type, or rather should likely use the (corrected) `..()` to do that.